src/cmd-handlers.ts

```diff
@@ -0,0 +1,41 @@
+import { Action } from './types'
+
+function cmd_handlers(reqFn: Action, args: any = {}, include?: string[]) {
+  async function load (this: any, msg: any) {
+    const repo_id = msg.q.repo_id
```

Comment:
seems like this still needs to be generalised?

Reply:
Thanks for the feedback ! 
Could you explain a bit more about ?  It seems like it needs some general "load" and "save" and so on. I can't find a intersection (without a lot of ifs) in the way each of them work, for example, "load" looks for msg.q, while when saving we'll be probably interested in entity attributes (msg.ent) and also we have to build a body from ent data. Thanks.